lightstep/sdk/metric/internal/viewstate/viewstate_test.go

```diff
@@ -1595,9 +1609,13 @@ func TestViewHints(t *testing.T) {
 			test.Point(seq.Start, seq.Now, minmaxsumcount.NewFloat64(inputs...), cumulative, set.ToSlice()...),
 		),
 		test.Instrument(
-			test.DescriptorDescUnit("gauge", sdkinstrument.SyncUpDownCounter, number.Float64Kind, "check it", ""),
+			test.DescriptorDescUnit("cumulative_gauge", sdkinstrument.SyncUpDownCounter, number.Float64Kind, "check it", ""),
```

Comment:
Is DefaultTemporality the same as CumulativeTemporality? i.e. should this test be called "default_gauge" since it doesn't provide `"temporality": "cumulative"`?

Reply:
Fixed.